### ISSUE TITLE
uploading data

### ISSUE THREAD

**Issue by scoriat** [jasp-stats/jasp-desktop#2099]
Bonjour, 
I am student doing a master in psychology. I am not a stats person. But I need to do a data analyse for my final project. I really like JASP as it is user-friendly and does great things. E.J. replying to emails from jasp.stats@gmail.com recommended I ask here for some help.

I can't figure out why I can't upload my data as csv. I have done different tests. 
I have two types of data files: one with text in the columns and one with values in the column.
 
The data with text is:
-       Frist column: manager or employee
-       Then each column is a different question (ex: 1. I find my work interesting) with 4 items (Linkert scale) possible in the column.
-       Some columns have Strongly Agree, Agree, Disagree or Strongly Disagree   
-       Other have Often, Occasionally, Rarely or Never

When I take only a small size of the data including all types of answers, 20 questions instead of the 84, it works fine. But I have to take the questions away and replace them by "Q1, Q2,Q3"  in the columns. 
[Values_numbers_10c.xlsx](https://github.com/jasp-stats/jasp-desktop/files/1430609/Values_numbers_10c.xlsx)
[Values_words_10c.xlsx](https://github.com/jasp-stats/jasp-desktop/files/1430610/Values_words_10c.xlsx)

Could you help please?
Thanks Sarah

**Comment by AlexanderLyNL** [comment]
Hi Sarah,

Here are two jasp and csv files. In both cases I saved the xlsx file as a csv file. In addition, I've removed the additional statistics you calculated in the number version. Does this answer your question?

[Sarah.zip](https://github.com/jasp-stats/jasp-desktop/files/1430705/Sarah.zip)

Cheers,
Alexander

**Comment by scoriat** [comment]
WOW AlexandreLyNL Thanks ! The text on works! What did you do? Just save them as CSV? why does it not work on my pc? 
I fell stupid there on the number version. But If I take out the stats on the full file it still does not work :-( 
If you still have time and can help...here it is. 
[Values_numbers.xlsx](https://github.com/jasp-stats/jasp-desktop/files/1431112/Values_numbers.xlsx)

**Comment by AlexanderLyNL** [comment]
Hi Sarah, 

You're correct. I opened the file in Excel and selected "Save as..." and choose as "format" "comma separated values". Here's the zip file. Please let me know whether it works. 

Cheers,
Alexander
[Sarah2.zip](https://github.com/jasp-stats/jasp-desktop/files/1432074/Sarah2.zip)

**Comment by AlexanderLyNL** [comment]
Hi Sarah, 

I'm closing this issue, because I think that it's solved. If that's not the case, don't hesitate to open this up again. Thanks for the feedback. 

Cheers,
Alexander

**Comment by scoriat** [comment]
Thanks a million! 
I am amazed by this! I did the same. I guess my pc is really bad and yours is amazing. I can finally start to work! My kids have seen me grummpy for more than a week now. I explained your help, they were impressed by how this works (helping freely poeple with your expdertise) and  they thank you too :-).
Yes this issue is now closed. all the best to you and wishing you a lovely day and life. 
Sarah
